app/i18n/el/feedback.php

```diff
@@ -94,6 +94,10 @@
 			'cache_cleared' => '<em>%s</em> cache has been cleared',	// TODO
 			'deleted' => 'Feed has been deleted',	// TODO
 			'error' => 'Feed cannot be updated',	// TODO
+			'favicon' => array(
+				'too_large' => 'Uploaded icon is too large. The maximum file size is <em>%s</em>',	// TODO
+				'unsupported_format' => 'Unsupported image file format',	// TODO
```

Comment:
```suggestion
				'too_large' => 'Uploaded icon is too large. The maximum file size is <em>%s</em>.',	// TODO
				'unsupported_format' => 'Unsupported image file format!',	// TODO
```

Maybe with some punctuation